rust/agents/relayer/src/checkpoint_relayer.rs

```diff
@@ -38,18 +40,18 @@ impl CheckpointRelayer {
     ) -> Self {
         let signed_checkpoint_gauge = leaf_index_gauge.with_label_values(&[
             "signed_offchain_checkpoint",
-            outbox.name(),
-            inbox_contracts.inbox.name(),
+            outbox.chain_name(),
```

Comment:
Actually relevant to some of what I was doing lately, is I wonder how we can get this better `chain_name()` to be used by the ethers-prometheus crate.